.github/workflows/e2e-nvidia-a10g-x1.yml

```diff
@@ -122,12 +122,12 @@ jobs:
           nvidia-smi
           python3.11 -m pip cache remove llama_cpp_python
 
-          CMAKE_ARGS="-DLLAMA_CUDA=on" python3.11 -m pip install .
+          CMAKE_ARGS="-DLLAMA_CUDA=on" python3.11 -m pip install -v .
```

Comment:
Can you add something to the description why the -v is being added

Reply:
Put a blurb in the main desc, this came from a suggestion @n1hility made on Slack